packages/sdk/typescript/human-protocol-sdk/package.json

```diff
@@ -14,7 +14,7 @@
     "clean": "rm -rf ./dist",
     "build": "npm run clean && tsc",
     "prepublish": "npm run build",
-    "test": "vitest -u test/storage.test.ts",
+    "test": "vitest -u test/staking.test.ts",
```

Comment:
Instead of changing the storage path, add your new path after it to execute both